README.md

```diff
@@ -1,10 +1,10 @@
 1. Replace `<your_account>` with your Github username in the link
-    - [DEMO LINK](https://<your_account>.github.io/js_task-move-spider-DOM/)
+    - [DEMO LINK](https://pdabizha.github.io/js_task-move-spider-DOM/)
```

Comment:
Add demo link to the PR description 
<img width="954" alt="image" src="https://github.com/user-attachments/assets/79fae1fc-3576-466e-ab2d-c836ef70a24b">